tsconfig.json

```diff
@@ -1,29 +1,21 @@
 {
-    "angularCompilerOptions": {
-        "strictMetadataEmit": true,
-        "skipTemplateCodegen": true,
-        "fullTemplateTypeCheck": true
-    },
-    "compilerOptions": {
-        "baseUrl": "",
-        "declaration": true,
-        "emitDecoratorMetadata": true,
-        "experimentalDecorators": true,
-        "lib": ["es2015", "dom"],
-        "mapRoot": "./",
-        "module": "es2015",
-        "moduleResolution": "node",
-        "outDir": "./dist",
-        "sourceMap": true,
-        "target": "es5",
-        "typeRoots": [
-            "./node_modules/@types"
-        ]
-    },
-    "files": [
-        "./src/index.ts"
+  "compileOnSave": false,
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "strict": false,
```

Comment:
Can you change this back to true and disable specific strict features, which are not currently supported instead? E.g. `strict: true, strictPropertyInitialization: false`.